src/main/java/frc/robot/commands/AlignCommand.java

```diff
@@ -31,21 +56,28 @@ public AlignCommand(Limelight limelight, DriveSubsystem driveSubsystem) {
                         EntryListenerFlags.kUpdate & EntryListenerFlags.kNew & EntryListenerFlags.kImmediate);
 
         // Degrees, degrees / second
-        aimyAim.setTolerance(3, 0.5);
-        aimyAim.enableContinuousInput(-180, 180);
-
-        addRequirements(driveSubsystem);
+        pid.setTolerance(3, 0.5);
+        pid.enableContinuousInput(-180, 180);
     }
 
     @Override
     public void execute() {
-        double output = aimyAim.calculate(limelight.getYaw(), 0);
+        // turning left is negative, right is positive
+        double offsetAngle;
+        if (useLimelight) offsetAngle = limelight.getYaw();
```

Comment:
After testing this on the robot, it appears as though the `limelight.getYaw()` needed to be negated.